libipl/p10/ipl0.C

```diff
@@ -277,6 +277,43 @@ static bool update_genesis_hwas_state(void)
 	return true;
 }
 
+/**
+ * @brief Wrapper function to execute continue mpipl on the pib
+ *
+ * @param[in] pib pib target to operate on
+ *
+ * return ipl error type enum
+ */
+static ipl_error_type  ipl_sbe_mpipl_continue(struct pdbg_target *pib)
+{
+	ipl_log(IPL_INFO, "Enter: ipl_sbe_mpipl_continue(%s)", pdbg_target_path(pib));
+
+	// Get the current SBE state
+	enum sbe_state state;
+
+	if (sbe_get_state(pib, &state)) {
```

Comment:
I usually avoid this pattern of using the return value of a function in a condition.  First call the function and then use the condition.  So when debugging you can inspect what was the return value of the function.